docs/content/guidelines/design.mdx

```diff
@@ -0,0 +1,300 @@
+---
+title: Design guidelines
+---
+
+import {Caption, Do, Dont} from '@primer/gatsby-theme-doctocat'
+import {Grid, Flex, Box} from '@primer/components'
+
+These are guidelines to use as a reference when designing new Octicons. If you're looking for guidance on how to use Octicons in your design, see our [Usage guidelines](/guidelines/usage).
+
+## Grid sizes
+
+Design two versions of each icon: a 16px version and a 24px version. 
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={4}>
+    <div>
+        <Flex bg="gray.1" justifyContent="center">
+            <img width="400" alt="" src="https://compai.pub/v1/png/41d55c2894e91bd7d0594d005ce51e2471292973343c01dcbbc271c34aa2d702"/>
+        </Flex>
+        <Caption>16x16 grid</Caption>
```

Comment:
Wondering if this caption is enough or if we should make sure there's alt tags in all the images.

Reply:
@emplums explained to me that if we have descriptive captions we should leave the `alt` attribute empty. From our [Caption docs](https://primer.style/doctocat/components/caption):
> Be sure to provide enough detail in your caption so users with assistive technology have adequate context. An empty alt tag alt="" should be used on the img in cases where the caption describes the image, thus making the image decorative in purpose.